### DISCUSSION TITLE
previous document on startup

### DISCUSSION THREAD

**Discussion by mil-ad** [ahrm/sioyek#691] [Q&A]
I don't have this problem on Linux but on macOS every time I open a document (cold open, i.e. sioyek isn't running before) the last_open_document also opens up and I end up with two windows. If I cold-open the prev document I get one window. Is this a bug?

I have `should_launch_new_window 1` in my config if that's relevant.

**Comment by ahrm** [comment]
Yes, this is a bug. I can't reproduce it on my systems though.

**Comment by mil-ad** [comment]
yeah doesn't happen on Linux but I'm using HEAD on Linux and 2.0.0 on macOS

**Comment by ahrm** [comment]
Maybe open an issue for this and the other problem and I will get to them when I have a chance to borrow a macbook.

**Comment by hacker-DOM** [comment]
I am encountering the same issue (MacOs). Especially annoying when screensharing in a professional setting lol

**Reply by ahrm** [reply]
Does it still happen on the preview release of the development branch here: https://github.com/ahrm/sioyek/releases/tag/sioyek3-alpha0?
